app/Platform/RouteServiceProvider.php

```diff
@@ -60,13 +69,16 @@ protected function mapWebRoutes(): void
             // Route::get('game/{game}/players', [GamePlayerController::class, 'index'])->name('game.player.index');
             Route::get('game/{game}/dev-interest', GameDevInterestController::class)->name('game.dev-interest');
 
+            // Route::get('game/{game}/hashes', [GameHashController::class, 'index'])->name('game.hashes'); "Supported Game Files"
```

Comment:
Heads up: initially the idea was to list all hashes, patches, save games at `game/{game}/assets` - not sure if a separate hashes page will be needed anymore then?

Reply:
Commented route removed in latest.